bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -49,34 +49,6 @@ metadata:
           },
           "spec": {}
         },
-        {
-          "apiVersion": "kuadrant.io/v1beta2",
-          "kind": "AuthPolicy",
-          "metadata": {
-            "name": "toystore"
-          },
-          "spec": {
-            "rules": {
-              "authentication": {
-                "apikey": {
-                  "apiKey": {
-                    "selector": {}
-                  }
-                },
-                "credentials": {
-                  "authorizationHeader": {
-                    "prefix": "APIKEY"
-                  }
-                }
-              }
-            },
-            "targetRef": {
-              "group": "gateway.networking.k8s.io",
-              "kind": "HTTPRoute",
-              "name": "toystore"
-            }
-          }
-        },
```

Comment:
Let's rename https://github.com/Kuadrant/kuadrant-operator/blob/1ac1104edae00a64cc3eae224f07fee46aadc190/config/samples/kuadrant_v1beta2_authpolicy.yaml to `kuadrant_v1beta3_authpolicy.yaml ` and add it to the samples path:
https://github.com/Kuadrant/kuadrant-operator/blob/1ac1104edae00a64cc3eae224f07fee46aadc190/config/samples/kustomization.yaml#L4
Since I feel like it's going to be missed if done later 🤔

Reply:
Ya sure, it is outside the doc guides and would be missed.